doc/source/config.rst

```diff
@@ -258,6 +258,23 @@ as part of the :func:`~satpy.modifiers.angles.get_angles` and
 used by multiple modifiers and composites including the default rayleigh
 correction.
 
+Clipping Negative Infrared Radiances
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Environment variable**: ``SATPY_CLIP_NEGATIVE_RADIANCES``
+* **YAML/Config Key**: ``clip_negative_radiances``
+* **Default**: False
+
+Whether to clip negative infrared radiances to the minimum possible value before
+computing the brightness temperature.
+If ``clip_negative_radiances=False``, pixels with negative radiances will have
+``np.nan`` brightness temperatures.
+
+Clipping of negative radiances is currently implemented for the following readers:
+
+* ``abi_l1b``
```

Comment:
This is just me brainstorming...but should we maybe had a `"readers"` sub-section in the configuration for things like this? @simonrp84 and @mraspaud what do you think?

So this new config key would be `satpy.config.set({"readers.clip_negative_radiances": True})` for example. Eh maybe it isn't worth making that distinction.

Reply:
Then the environment variables would be `SATPY_READERS.CLIP_NEGATIVE_RADIANCES`? The dot is allowed?

Reply:
`SATPY_READERS__CLIP_NEGATIVE_RADIANCES` (two underscores where the dot is).

Reply:
I agree, this is really for readers...

Reply:
@mraspaud @djhoese do you want that I rename the config key as `"readers.clip_negative_radiances"`?

Reply:
yes, please :+1: